editor/src/clj/editor/atlas.clj

```diff
@@ -843,8 +844,8 @@
   
   (output texture-set-pb   g/Any               :cached produce-atlas-texture-set-pb)
 
-  (output aabb             AABB                (g/fnk [layout-size]
-                                                 (if (= [0 0] layout-size)
+  (output aabb             AABB                (g/fnk [layout-size layout-rects]
+                                                 (if (or (= [0 0] layout-size) (empty? layout-rects))
```

Comment:
If there are no images in atlas, we should use one of predefined AABBs (empty or null)